### ISSUE TITLE
Document how to rotate generated secrets

### ISSUE THREAD

**Issue by jandubois** [cloudfoundry-incubator/kubecf#703]
There is a documentation page for it already: https://github.com/cloudfoundry-incubator/kubecf/blob/master/doc/secret_rotation.md

However, it only documents how to rotate the database encryption key, which is just one subtopic of rotating all the secrets in the cluster.

The rotation of generated secrets will have to be managed by cf-operator, but we need documentation about how to trigger it.

**Comment by viovanov** [comment]
There should be a script (like a one line) to perform rotation for all secrets.

**Comment by viovanov** [comment]
Also see the following resources:

https://quarks.suse.dev/docs/quarks-secret/development/#_secretrotation-controller_
https://quarks.suse.dev/docs/quarks-secret/#rotate-credentials

**Comment by mook-as** [comment]
@jandubois Is this already done by #854? That documented how to rotate a QuarksSecret.  It doesn't contain a script to rotate all secrets, though.

**Comment by viovanov** [comment]
@mook-as I don't think that's enough, and a sample script should be part of this

**Comment by mudler** [comment]
Can we please avoid to document this inside the kubecf repo, but to https://kubecf.suse.dev/ instead? It sounds more user-facing docs

**Comment by mook-as** [comment]
Yep, this will be an update to https://github.com/cloudfoundry-incubator/kubecf-docs/blob/master/content/en/docs/Tasks/secrets.md

But to make it _reasonable_ for the user, I'll attempt to make a QuarksJob to automate the rotate-all-secrets case, and tell the user to trigger that instead.
